numpy/random/_generator.pyx

```diff
@@ -454,6 +454,21 @@ cdef class Generator:
         >>> x-y
         0.08 # may vary
 
+        The distribution can be visualized as follows:
+
+        >>> import numpy as np
+        >>> import matplotlib.pyplot as plt
+        >>> import scipy.stats as stats
+        >>> l = 2
+        >>> rng = np.random.default_rng()
+        >>> sample = rng.exponential(scale=l, size=1000)
+        >>> count, bins, ignored = plt.hist(sample, 30, density=True)
```

Comment:
```suggestion
        >>> count, bins, _ = plt.hist(sample, 30, density=True)
```
The _ is commonly used to ignore an unused output

Reply:
This is a good idea and should be changed in a few other places as well!